BentoKit/BentoKit/Screen/Screen.swift

```diff
@@ -22,10 +22,34 @@ public struct Screen<SectionId: Hashable, RowId: Hashable> {
         box: Box<SectionId, RowId>,
         pinnedToTopBox: Box<SectionId, RowId> = .empty,
         pinnedToBottomBox: Box<SectionId, RowId> = .empty
+    ) {
+        self.init(
+            titleItem: .text(title),
+            leftBarItems: leftBarItems,
+            rightBarItems: rightBarItems,
+            formStyle: formStyle,
+            focusMode: focusMode,
+            shouldUseSystemSeparators: shouldUseSystemSeparators,
+            box: box,
+            pinnedToTopBox: pinnedToTopBox,
+            pinnedToBottomBox: pinnedToBottomBox
+        )
+    }
+
+    public init(
+        titleItem: NavigationTitleItem = .text(""),
```

Comment:
Can we please change this to make it explicit for consumers they need to pass a `title` or `titleItem`? Having an empty one should be defined by them not us.

Reply:
I can make `titleItem` have no default value, but `title` (in the old initialiser) always had a default value. Removing it now would break a lot of existing code.

Reply:
I still think it should be done, this needs to be defined explicitly and not implicitly.

Reply:
Done 💥

Reply:
❤️